### PR TITLE
226 227 restructure learning objectives

### PR DESCRIPTION
**Pull request by drelliche**
This addresses the parallel concerns raised in https://github.com/arcus/education_modules/issues/226 and https://github.com/arcus/education_modules/issues/227 

Namely: the idea of writing to a file is introduced for the first time in the Bash 101 module, then more expansively tackled in the "combining commands" module. This is now explicitly acknowledged in both places and learning objectives have been updated as well.

### CODE REVIEW

File: bash_103_combining_commands/bash_103_combining_commands.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 author:   Elizabeth Drellich and Nicole Feldman
 email:    drelliche@chop.edu and feldmanna@chop.edu
-version: 1.1.0
+version: 1.3.0
```
Comment (rosemm) [inline bash_103_combining_commands/bash_103_combining_commands.md:5]:
```suggestion
version: 1.2.0
```

This should just be incremented by 1, right?

**Review by rosemm — CHANGES_REQUESTED** [review]
One little typo (I think), otherwise LGTM!

File: bash_103_combining_commands/bash_103_combining_commands.md (hunk 1 above)
Comment (drelliche) [reply bash_103_combining_commands/bash_103_combining_commands.md:5]: I think it has already been incremented in another PR, which is why there is a conflict... current version on main is 1.2.0

Comment (rosemm) [reply bash_103_combining_commands/bash_103_combining_commands.md:5]: Ah makes sense!

Comment (rosemm) [reply bash_103_combining_commands/bash_103_combining_commands.md:5]: Ah makes sense!

**Review by rosemm — APPROVED** [review]
lgtm

**Pull request merged**